README.md

```diff
@@ -20,8 +20,7 @@ This action can be used to deploy Azure Resource Manager templates at different
 * `parameters`: Specify the path or URL to the Azure Resource Manager deployment parameter values. Or local / remote value file.  
 * `deploymentMode`: `Incremental`(default) (only add resources to resource group) or `Complete` (remove extra resources from resource group) or `Validate` (only validates the template). 
 * `deploymentName`: Specifies the name of the resource group deployment to create.
-
-
+* `failOnStdError`: Specify whether to fail the action when std error is not null.
```

Comment:
Rather than saying not null, say if the some data is written to stderr stream of az cli